capstone/src/main/webapp/style.css

```diff
@@ -0,0 +1,82 @@
+
+* {
+  box-sizing: border-box;
+}
+
+.column {
+    float: left;
+    padding: 10px;
+}
+
+.team-builder{
+  left: 0;
+  width: 75%;
+}
+
+.searchbar {
+  right: 0;
+  width: 25%;
+}
+
+.left{ 
+  background-color:#aaa;
+  width: 25%;
+}
+
+.right {
+  background-color:#bbb;
+  width: 75%;
+}
+
+.row:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+.chart {
+  border: 1px solid;
+  float: left;
+  min-height: 90px;
+  padding: 10px;
+}
+
+
+#myInput {
+  background-image: url('images/searchicon.png'); /* Add a search icon to input */
+  background-position: 10px 12px; /* Position the search icon */
+  background-repeat: no-repeat; /* Do not repeat the icon image */
+  width: 100%; /* Full-width */
+  font-size: 16px; /* Increase font-size */
+  padding: 12px 20px 12px 40px; /* Add some padding */
+  border: 1px solid #ddd; /* Add a grey border */
+  margin-bottom: 12px; /* Add some space below the input */
+}
+  
+#myUL {
+  /* Remove default list styling */
```

Comment:
more descriptive names, want to know what every item does... maybe like playerUL or something?